proto/vulcanize/registry/v1beta1/attributes.proto

```diff
@@ -30,4 +32,172 @@ message WebsiteRegistrationRecord {
   string tls_cert_cid       = 4 [(gogoproto.moretags) = "json:\"TLSCertCID\" yaml:\"TLSCertCID\""];
   string type               = 5 [(gogoproto.moretags) = "json:\"type\" yaml:\"type\""];
   string version            = 6 [(gogoproto.moretags) = "json:\"version\" yaml:\"version\""];
+}
+
+message GitRepository{
+  string name =1  [(gogoproto.moretags) = "json:\"name\" yaml:\"name\""];
+  string repo_reference=2 [(gogoproto.moretags) = "json:\"repo_reference\" yaml:\"repo_reference\""];
+  string description=3  [(gogoproto.moretags) = "json:\"description\" yaml:\"description\""];
+  string version            = 4 [(gogoproto.moretags) = "json:\"version\" yaml:\"version\""];
+}
+
+message Binary{
+  string raw_multicodec=1;
+  string raw_binary=2;
+  string targeted_arch=3;
+  string runtime_version=4;
+  string repo_reference=5;
+  string version=6;
+  string type=7;
+}
+
+message DockerImage{
+  string image_id=1;
+  string image_binary=2;
+  string reference_repo=3;
+  string version=4;
+  string type=5;
+}
+
+message WatcherRegistrationRecord{
+  WatcherMetadata metadata =1;
+  string repo_reference=2;
+  WASMBinary wasm=3;
+  string version=4;
+  string type=5;
+
+  message WatcherMetadata{
+    string version=1;
+    string chain_reference=2;
+  }
+
+  message WASMBinary{
+    string reference=1;
+    WASMBinaryMetadata metadata=2;
+  }
+  
+  message WASMBinaryMetadata{
+    string compiler_version=1;
+    string execution_engine_version=2;
+  }
+}
+
+message ResponderContract{
+  string service_provider_ref=1;
+  string auction_ref=2;
+  string watcher_ref=3;
+  string version=4;
+  string type=5;
+}
+
+message NPMPackage{
+  string repo_reference=1;
+  string npm_package_ref=2;
+  string version=3;
+  string type=4;
+}
+
+message ChainRegistrationRecord{
```

Comment:
Let's add fields:

`string chain_id`
`string network_id`
`string genesis_hash`

Reply:
is the genesis_hash a CID reference or just a hash string?

Reply:
Good question... let's go with full CID for now